### DISCUSSION TITLE
[archived] "High Availability" docs discussion

### DISCUSSION THREAD

**Discussion by giscus** [argoproj/argo-workflows#10442] [General]
# argo-workflows/high-availability/

https://argoproj.github.io/argo-workflows/high-availability/

**Comment by alejoGT1202** [comment]
Hello, I've a question regarding the HA what does it mean `one of which will be kept on hot-standby`. 

How should I enable HA to my workflow-controller, should I set the number of replicas to 2, and create a PDB with `minAvailable` as 1?

**Reply by nicolas-vivot** [reply]
Yes. I would advise you to use `maxUnavailable` with value `1` instead of `minAvailable` so that if you increase the number of replicas, that would ensure a better HA.

**Comment by zhongwuso** [comment]
hello, i just  set the number of replicas to 2 to my workflow-controller, and  then i saw that  works well, is that ok?

**Reply by zhongwuso** [reply]
My k8s version is 1.18 , my workflow-controller version is 3.1.15

**Reply by agilgur5** [reply]
First off, those are very old and unsupported k8s and Argo versions, so I think anyone would suggest that you update to supported versions.

Second, yes, this exact docs page says that Argo versions >3.0 support a hot-standby for the Controller. There are some bugs with it in earlier versions though.

**Reply by zhongwuso** [reply]
thanks for your reply ! i am going to update k8s version
